src/machine/machined-varlink.c

```diff
@@ -781,6 +781,8 @@ static int manager_varlink_init_machine(Manager *m) {
                         "io.systemd.Machine.Unregister",  vl_method_unregister,
                         "io.systemd.Machine.Terminate",   vl_method_terminate,
                         "io.systemd.Machine.Kill",        vl_method_kill,
+                        "io.systemd.Machine.CopyFrom",    vl_method_copy_from,
+                        "io.systemd.Machine.CopyTo",      vl_method_copy_to,
```

Comment:
you are taking a function pointer of an inline function here. which i guess makes the C compiler actually synthesize a non-inline version of your function. but given you don't need it otherwise, maybe make the inline functions just regilar ones, and then make the implementation function static